crates/legacy/src/builder_state.rs

```diff
@@ -661,6 +662,13 @@ impl<Types: NodeType> BuilderState<Types> {
 
         for tx in txn_commitments.iter() {
             self.txns_in_queue.remove(tx);
+            // set the status of tx as sequenced, this doesn't mean tx is already decided, but it means tx is included in a builder state, it will either be sequenced or skipped
```

Comment:
Why do we want to say it's sequenced when it's not decided yet?

Reply:
See my reply below.